addons/populate/functions/fnc_createPatrols.sqf

```diff
@@ -0,0 +1,57 @@
+#include "script_component.hpp"
+/*
+    Author:
+        Bridg
+
+    Description:
+        Spawns a unit at the passed position
+
+    Parameters:
+        0: _numberOfUnitsToSpawn <NUMBER>
+        1: _module <OBJECT>
+        2: _group <GROUP>
+
+    Return value:
+        Nothing
+*/
+
+params ["_numberOfUnitsToSpawn", "_side", "_module", "_unitPoolArray"];
+
+private _group = grpNull;
+private _currenGrouptUnitCount = 0;
+private _patrolSize = _module getVariable [QGVAR(patrolSize), 0];
```

Comment:
`0` is a bad default, should exit if value from the module is 0 otherwise the patrols will never be tasked